### ISSUE TITLE
Claim process didn't work

### ISSUE THREAD

**Issue by parasyte** [linuxserver/docker-plex#235]
## Expected Behavior

Signing into the Plex web app for the first time should walk you through the claim process and setup wizard.

## Current Behavior

After signing in, I was left with a useless web client that didn't know it was server. It didn't matter how many times I tried to sign out and back in. I even removed all of my shared libraries in hopes that the UI was just messed up. That left me with the dreaded "Get Plex Media Server" button that everyone sees and no one knows what to actually do about.

To fix this, I had to:

1. Sign out of the web app
2. Add a password to my account through plex.tv (I have only used Google to sign in, up to this point)
3. Sign out of plex.tv
4. Run [this script](https://github.com/ukdtom/ClaimIt/blob/master/linux/claimpms.sh) to claim the server

After that, signing in brought up the claim and setup wizard as expected.

## Steps to Reproduce

1. Sign in for the first time after a fresh install (I used Google)
2. Notice that the web app shows all servers you already have shared, but not this new server. No option to claim it, either.
3. Search the web for 5 hours until you find the script linked above to fix it.
4. ???
5. Profit!

## Environment
**OS:**      LibreElec
**CPU architecture:** arm64
**How docker service was installed:**      

Installed through the normal LibreElec add-on repo.

The `/data/movies` volume is a CIFS mount, but that doesn't really matter. The files were all readable from within the container (`docker exec -it plex bash` and `cat` some files in `/data/movies` for fun...) and even an empty movies directory should not cause the Plex Media Server to be completely oblivious that it needs to be configured.

## Command used to create docker container (run/create/compose/screenshot)

Just using the systemd unit that comes with the LibreElec addon. My CIFS mount is another systemd unit with a `Before=docker.linuxserver.plex.service` setting. Again, this shouldn't matter, since it only ensures files are visible to the container. And this was never a problem.

## Docker logs

<details><summary>Click for boring useless logs</summary>

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 01-envfile: executing...
[cont-init.d] 01-envfile: exited 0.
[cont-init.d] 10-adduser: executing...
-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/
Brought to you by linuxserver.io
-------------------------------------
To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------
User uid:    65534
User gid:    65534
-------------------------------------
[cont-init.d] 10-adduser: exited 0.
[cont-init.d] 40-chown-files: executing...
[cont-init.d] 40-chown-files: exited 0.
[cont-init.d] 45-plex-claim: executing...
[cont-init.d] 45-plex-claim: exited 0.
[cont-init.d] 50-gid-video: executing...
[cont-init.d] 50-gid-video: exited 0.
[cont-init.d] 60-plex-update: executing...
#####################################################
#   Login via the webui at http://<ip>:32400/web    #
#   and restart the docker, because there was no    #
#      plex token found in the preference file      #
#####################################################
[cont-init.d] 60-plex-update: exited 0.
[cont-init.d] 99-custom-scripts: executing...
[custom-init] no custom files found exiting...
[cont-init.d] 99-custom-scripts: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
Starting Plex Media Server.
```
</details>

And FWIW, there is now a token in the preference file, but only after I ran the claim script that fixed everything. This was the old prefs file, pre-claim-script:

```
root@Plex-LibreELEC:/# cat '/config/Library/Application Support/Plex Media Server/Preferences.xml'
<?xml version="1.0" encoding="utf-8"?>
<Preferences OldestPreviousVersion="1.20.1.3252-a78fef9a9" MachineIdentifier="..." ProcessedMachineIdentifier="..." AnonymousMachineIdentifier="..." MetricsEpoch="1"/>
```

**Comment by chris-kahn** [comment]
I have the same problem only that script doesn't work, I just get a fatal error 401 when sending POST request to claim the server.

**Comment by parasyte** [comment]
That's an authentication error. Check or change your password? My account was originally logged in with google, and I had to add a password to my account to use the script.

**Comment by aptalca** [comment]
In the original post, the claim process never ran, which means either the plex claim env var was not set properly, or there was already a plex online token in the preferences.xml.

I'm closing this issue as the discussion evolved into troubleshooting a 3rd party script we do not support. Feel free to open a new issue for help with this image, but make sure you fill out the template fully (including docker create/run or compose yaml)

**Comment by parasyte** [comment]
@aptalca The original preferences.xml is included in the OP. No token or other claims were set there.

I'm not sure what environment variable you are talking about, but it wouldn't have been anything that I configured.

**Comment by aptalca** [comment]
The readme explains it. If you use host networking, you can get the wizard as long as you connect from the same network subnet. If not, you need to use the plex claim env var

**Comment by chris-kahn** [comment]
I used the `PLEX_CLAIM` environment variable and while it said that the server claim process works, actually accessing my server setup wizard still didn't work, I was stuck with the `No soup for you` message when navigating to `<your-ip>:32400/web`. It seems there are CORS errors because some requests try to connect to `127.0.0.1:32400` rather than my server's IP and gets rejected by the CORS policy.

The only way I could finally get it to work was port tunnelling with SSH: `ssh myserver -L 32400:localhost:32400` and then going to `127.0.0.1:32400/web`. After doing that, the setup is complete and I can now access my server from my Plex apps.

**Comment by AnnoyedDev** [comment]
i had the same issue. Tunneling was the way to fix the problem.
